gplugins/klayout/get_density.py

```diff
@@ -41,6 +41,7 @@ def put(
 def calculate_density(
     gdspath: Path,
     layer: tuple[int, int],
+    cellname: str | None = None,
```

Comment:
should cellname be `None` or an empty string? I genuinely don't know, just wary of setting `str`s to `None`